e2e_tests/docker/indexer/Dockerfile

```
@@ -22,7 +22,7 @@ WORKDIR /opt/go/indexer
 RUN make
 ENV PATH="${HOME}/go/bin/:${PATH}"
 WORKDIR /opt/go
-RUN git clone https://github.com/algorand/conduit
+RUN git clone -b 23may-indexer-import-types https://github.com/algorand/conduit
```

TODO: don't merge with this branch

```suggestion
RUN git clone -b master https://github.com/algorand/conduit
```